fairlearn/__init__.py

```diff
@@ -9,7 +9,7 @@
 from .show_versions import show_versions  # noqa: F401
 
 __name__ = "fairlearn"
-__version__ = "0.6.1.dev0"
+__version__ = "0.6.2.dev0"
```

Comment:
This whole PR seems to need to be cherry picked for the release branch, and therefor this change needs to be in a separate PR I think.

Reply:
This fix and the changes to `index.html` need to be on `main`, since the docs are only build from `main`. The subsequent PR (#740 ) will get cherry picked, since otherwise the release breaks.

Reply:
hmm, ok.

Reply:
I'd be open to improved ways of dealing with the versioned docs. I do take your point that this update looks slightly odd.